test/conversions.jl

```diff
@@ -6,28 +6,34 @@ warsaw = first(compile("Europe/Warsaw", tzdata["europe"]))
 apia = first(compile("Pacific/Apia", tzdata["australasia"]))
 midway = first(compile("Pacific/Midway", tzdata["australasia"]))
 
-# Converting a ZonedDateTime into a DateTime
-dt = DateTime(2015, 1, 1, 0)
-zdt = ZonedDateTime(dt, warsaw)
-@test DateTime(zdt) == dt
-
-# Converting from ZonedDateTime to DateTime isn't possible as it is always inexact.
-@test_throws MethodError convert(DateTime, zdt)
-
-# Vectorized accessors
-n = 10
-arr = fill(zdt, n)
-@test Dates.DateTime.(arr) == fill(dt, n)
```

Comment:
we don't want this test anymore?

Reply:
Isn't this just testing that broadcasting works?
Testing that broadcasting works on arrays is not our problem.

This test is leftover from ancient times  (https://github.com/JuliaTime/TimeZones.jl/pull/7)
when `DateTime(::Vector{ZonedDateTime})` was defined.
And so needed tests.
But we got rid of that because julia gained broadcasting,
and I guess the test just got converted over to use the new broadcasting,
without realizing that the code it was suppose to test didn't exist anymore.

Reply:
We can drop this code now